tests/test_schneiderelectric.py

```diff
@@ -1,145 +1,14 @@
-                "out_clusters": [
-                    "0x0003",
-                    "0x0005",
-                    "0x0006",
-                    "0x0008",
-                    "0x0019",
-                    "0x0102",
-                ],
-            },
-        },
-        "manufacturer": "Schneider Electric",
-        "model": "1GANG/SHUTTER/1",
-        "class": "zigpy.device.Device",
-    }
-    assert_signature_matches_quirk(
-        zhaquirks.schneiderelectric.shutters.OneGangShutter1, signature
-    )
-
-
-async def test_1gang_shutter_1_go_to_lift_percentage_cmd(zigpy_device_from_quirk):
-    """Asserts that the go_to_lift_percentage command inverts the percentage value."""
-
-    device = zigpy_device_from_quirk(
-        zhaquirks.schneiderelectric.shutters.OneGangShutter1
-    )
-    window_covering_cluster = device.endpoints[5].window_covering
-
-    p = mock.patch.object(window_covering_cluster, "request", mock.AsyncMock())
-    with p as request_mock:
-        request_mock.return_value = (foundation.Status.SUCCESS, "done")
-
-        await window_covering_cluster.go_to_lift_percentage(58)
-
-        assert request_mock.call_count == 1
-        assert request_mock.call_args[0][1] == (
-            WindowCovering.ServerCommandDefs.go_to_lift_percentage.id
-        )
-        assert request_mock.call_args[0][3] == 42  # 100 - 58
-
-
-async def test_1gang_shutter_1_unpatched_cmd(zigpy_device_from_quirk):
-    """Asserts that unpatched ZCL commands keep working."""
-
-    device = zigpy_device_from_quirk(
-        zhaquirks.schneiderelectric.shutters.OneGangShutter1
-    )
-    window_covering_cluster = device.endpoints[5].window_covering
-
-    p = mock.patch.object(window_covering_cluster, "request", mock.AsyncMock())
-    with p as request_mock:
-        request_mock.return_value = (foundation.Status.SUCCESS, "done")
-
-        await window_covering_cluster.up_open()
-
-        assert request_mock.call_count == 1
-        assert request_mock.call_args[0][1] == (
-            WindowCovering.ServerCommandDefs.up_open.id
-        )
-
-
-async def test_1gang_shutter_1_lift_percentage_updates(zigpy_device_from_quirk):
-    """Asserts that updates to the ``current_position_lift_percentage`` attribute.
-
-    (e.g., by the device) invert the reported percentage value.
-    """
-
-    device = zigpy_device_from_quirk(
-        zhaquirks.schneiderelectric.shutters.OneGangShutter1
-    )
-    window_covering_cluster = device.endpoints[5].window_covering
-    cluster_listener = ClusterListener(window_covering_cluster)
-
-    window_covering_cluster.update_attribute(
-        WindowCovering.AttributeDefs.current_position_lift_percentage.id,
-        77,
-    )
-
-    assert len(cluster_listener.attribute_updates) == 1
-    assert cluster_listener.attribute_updates[0] == (
-        WindowCovering.AttributeDefs.current_position_lift_percentage.id,
-        23,  # 100 - 77
-    )
-    assert len(cluster_listener.cluster_commands) == 0
```

Comment:
Ideally, we should keep the tests (except for the signature test). You should be able to use `zigpy_device_from_v2_quirk` for the tests. Might need to search the repo for some examples.

Reply:
Good point, done.